otter/auth.py

```diff
@@ -235,6 +235,12 @@ def __init__(self, identity_admin_user, identity_admin_password, url, admin_url)
 
     @wait(ignore_kwargs=['log'])
     def _auth_me(self, log=None):
+        def _log_failed_auth(err):
+            """
+            Log this as a string we know we can find in the logging feed
+            """
+            log.err('Failed to do identity impersonation')
```

Comment:
I suggest adding an `otter_msg_type='impersonation-failed'`, to give us a nicer identifier to filter on (in addition to the English message you have now).